scripts/validate-terraform.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# This script works from stdin and expects one filename per line.
+# To call it, e.g.
+# find ./website/docs -type f \( -name '*.md' -o -name '*.markdown' \) \
+#   | ./scripts/validate-terraform.sh
+
+TERRAFMT_CMD="terrafmt"
+if [ -f ~/developer/terrafmt/terrafmt ]; then TERRAFMT_CMD="$HOME/developer/terrafmt/terrafmt"; fi
+
+exit_code=0
+
+# Configure the rules for tflint.
+# The *_invalid_* rules disabled here prevent evaluation of expressions.
+rules=(
+    # Syntax checks
+    "--enable-rule=terraform_deprecated_interpolation"
+    "--enable-rule=terraform_deprecated_index"
+    "--enable-rule=terraform_comment_syntax"
+    # Ensure modern instance types
+    "--enable-rule=aws_instance_previous_type"
+    "--enable-rule=aws_db_instance_previous_type"
+    "--enable-rule=aws_elasticache_cluster_previous_type"
+    # Prevent some configuration errors
+    "--enable-rule=aws_route_specified_multiple_targets"
+    # Prevent expression evaluation
```

Comment:
Nit: This looks like it might be hard for contributors and maintainers to understand/maintain -- is there anything we can do to reduce the need for single shot disabling rules like these?

Reply:
Once `tflint` 0.20 is released, it will have the option to only run specified rules https://github.com/terraform-linters/tflint/pull/875